### PR TITLE
Willys/back-to-basics-oss

### PR DESCRIPTION
**Pull request by wpschaub**
An open source awareness post to gibe Andreas an open door to delve into and explore blueprint and TFM topics in detail.

The TBD placeholders will be updated during publication merge. This post is dependent on Andreas' bio post.

### CODE REVIEW

**Review by danielrbroderick — DISMISSED** [review]

File: content/back-to-basics-oss.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+# Azure DevOps Automation Pipeline Templates
+
+[Pipelines - Meet our second-generation app-type blueprints](https://wsbctechnicalblog.github.io/yaml-pipelines-part10.html) touches on the core rationale behind our investment in this OSS project. When tasked with creating a continuous delivery pipeline, soliciting input from 13 software engineers is akin to inviting a multitude of perspectives, potentially resulting in more than 13 different approaches. While this diversity underscores autonomy and fosters innovation, it can inadvertently divert our delivery teams from their primary obligation of consistently delivering functional code and value. 
+
+>
+> ![Pipeline status](/images/back-to-basics-oss-1.png)
+>
+
+Moreover, the drift and sprawl of pipeline configuration snowflakes complicates reuse and inflates support and maintenance overheads and cst of ownership. Analogous to architectural blueprints, our standardized pipeline templates—both generic and application-specific—facilitate the swift and uniform creation of countless delivery pipelines. This approach champions:
```
Comment (MerlinDarkstar) [inline content/back-to-basics-oss.md]: cst s/b cost

Comment (MerlinDarkstar) [reply content/back-to-basics-oss.md]: Also, I think there should be a space in here "application-specific-facilitate" s/b "application-specific facilitates"

File: content/back-to-basics-oss.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- **Security**: By integrating security best practices into our pipeline blueprints, we embed security measures directly into the software delivery process. This proactive approach helps identify and mitigate security vulnerabilities early in the development cycle, enhancing the overall resilience and robustness of our applications.
+
+>
+> ![Callout](/images/back-to-basics-batch-size-alert.png)
+>
+> **EXPLORE** our [Application-type CI/CD blueprint template](https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Templates.v2) OSS project, collaborate, and continuously improve the templates with us. 
+>
+
+To get an insight into WHY we engineered these blueprints, HOW they evolved, and HOW we use them, please refer to the [Part 1: Pipelines - Why bother and what are our nightmares and options?](/why-pipelines-part1.html) series. We are also working on a v2 blueprint cookbook, which we may self-publish this year - watch the space.
```
Comment (MerlinDarkstar) [inline content/back-to-basics-oss.md]: "watch the space" s/b "watch this space"

File: content/back-to-basics-oss.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+---
+
+# Technology Framework Monitor (TFM)
+
+The Technology Framework Monitor project is a side project by [Andreas Mertens](TBD), who wanted to help us find out the use of outdated versions of .NET that are used by solutions in hundreds of Azure Repos. The TFM product will go through and collect information on every project in an Azure Developer Operations organization, or repository, check the files inside, and then save the data collected into a Microsoft Standard Query Language (SQL server database.
```
Comment (MerlinDarkstar) [inline content/back-to-basics-oss.md]: missing closing bracket on (SQL

**Review by MerlinDarkstar — DISMISSED** [review]

File: content/back-to-basics-oss.md (hunk 2 above)
Comment (wpschaub) [reply content/back-to-basics-oss.md]: Fixed.

File: content/back-to-basics-oss.md (hunk 1 above)
Comment (wpschaub) [reply content/back-to-basics-oss.md]: Fixed

**Review by danielrbroderick — DISMISSED** [review]

File: content/back-to-basics-oss.md (hunk 3 above)
Comment (wpschaub) [reply content/back-to-basics-oss.md]: Fixed

**Pull request merged**